LDAP Module Documentation (#5969)
mendix/docs: content/en/docs/appstore/modules/ldap.md

---
title: "LDAP"
url: /appstore/modules/ldap/
category: "Modules"
description: "Describes the configuration and usage of the LDAP module, which is available in the Mendix Marketplace."
tags: ["marketplace", "marketplace component", "ldap", "IdP", "identity provider", "platform support"]
##If moving or renaming this doc file, implement a temporary redirect and let the respective team know they should update the URL in the marketplace.
---

## 1 Introduction

> The LDAP module is a client-side implementation of the Lightweight Directory Access Protocol (LDAP) to allow your app to communicate with an LDAP server. It can be used in a Mendix app to synchronize end-users, their group memberships, and their status from an LDAP server such as Microsoft AD.

Author: Need a link to the marketplace.

    * **Import users** – Import and synchronize the end-users specified the configuration. This will make the end-user information available in your Mendix app.
    * **Only authenticate users** – This will only authenticate existing Mendix end-users against the LDAP server, but it will not synchronize any information. If an end-user is not known in Mendix, they cannot sign in.
    * **Authenticate and create** – This will not synchronize end-users. However, if an end-user that is unknown in Mendix signs in using valid LDAP authentication, a Mendix end-user will be created, and the end-user info will be copied from the LDAP server at that moment.

* **Map users to** – This specifies which entity type objects should be created when creating new end-users. You can choose from all the specializations of **System.User**. For example, you may want to use the `Administration.Account` entity if you are using the Administration module.

> * **Domain suffix** – This is a concatenation of the LDAP root directory fields. It is used to authenticate end-users and will be concatenated to the Mendix user name to form the LDAP user name. At Mendix, this results in `@mendixdomain.local`.

Author: Is this relevant to other customers, or is it an example taken from our own internal configuration? Are there any security issues in publishing this?
```suggestion
* **Domain suffix** – This is a concatenation of the LDAP root directory fields. It is used to authenticate users and will be concatenated to the Mendix user name to form the LDAP user name. For example, the domain suffix at Mendix is `@mendixdomain.local`.
```

Reviewer: yes this is relevant to other customers.
I don't think any security issues in publishing this.
